packages/boxel-ui/addon/components/dropdown-button/index.gts

```diff
@@ -0,0 +1,93 @@
+import type { TemplateOnlyComponent } from '@ember/component/template-only';
+import { type WithBoundArgs } from '@glint/template';
+import { hash } from '@ember/helper';
+import BoxelDropdown from '../dropdown';
+import BoxelMenu from '../menu';
+import cn from '@cardstack/boxel-ui/helpers/cn';
+import { svgJar } from '@cardstack/boxel-ui/helpers/svg-jar';
+
+interface Signature {
+  Element: HTMLButtonElement;
+  Args: {
+    icon: string;
+    label: string;
+    class?: string;
+    contentClass?: string;
+    noHoverStyle?: boolean;
```

Comment:
What does this one mean?

Reply:
I see that we've used this arg in the cardstack monorepo where we used this component, but it looks like it doesn't actually do anything. I'll remove it